LICENSE

```diff
@@ -1,23 +1,47 @@
-The MIT License (MIT)
```

Comment:
Since there was some extra wording added I wasn't sure whether we could keep this just saying it's the standard MIT license text. It's the same clauses of course but It's not a straight-text copy of the MIT license. I'll pull in @stevenayr for an opinion here. Note that the code file still does say MIT License.